src/pages/settings/AboutPage/AboutPage.js

```diff
@@ -95,64 +96,60 @@ function AboutPage(props) {
         }));
     }, [translate, waitForNavigate]);
 
+    const overlayContent = useCallback(
+        () => (
+            <View style={[styles.pAbsolute, styles.w100, styles.h100, styles.justifyContentEnd, styles.pb5]}>
+                <Text
+                    selectable
+                    style={[styles.textLabel, styles.textIvoryLight, styles.alignSelfCenter]}
+                >
+                    v{Environment.isInternalTestBuild() ? `${pkg.version} PR:${CONST.PULL_REQUEST_NUMBER}${getFlavor()}` : `${pkg.version}${getFlavor()}`}
+                </Text>
+            </View>
+        ),
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Add a comment explaining the reason

Reply:
I am not sure whether we require comment here bcoz isn't useCallback without dependencies descriptive enough to only create a function on the mount?

Reply:
I meant on why we are disabling the lint rule e.g. you can just write `styles do not change`